python/psautohint/otfFont.py

```diff
@@ -652,24 +654,37 @@ def convertBezToT2(bezString):
     return t2Program
 
 
+def _run_tx(args, font_format):
+    try:
+        subprocess.check_call(["tx"] + args)
+    except subprocess.CalledProcessError as e:
+        raise FontParseError(e)
+    except OSError:
```

Comment:
Thinking about this some more, I think this `except` should be moved to a new method (`_check_tx`?) that is called much earlier, preferably in `__main__.get_options`, right after `_validate_font_paths` is called. Then we can simply throw a `parser.error`.
Also, I think the error message should say something about how to get **tx**.

`NotImplementedError` (in **__main__.py**) won't be necessary anymore once the changes above are made.